example/test_tetanus.py

```diff
@@ -0,0 +1,115 @@
+"""Tetanus spec tests."""
+from typing import List
+
+import asyncio
+import pytest
+
+from Lego3.example.components.giraffe import Giraffe
+from Lego3.example.libs.tetanus import Tetanus
+
+
+TOOL = 'ncat -l {} --keep-open --udp --exec "/bin/cat"'
+BUGY_LOGS_TOOL = TOOL + ' --output log.txt'
+BUGY_SEND_TOOL = 'ncat -l {} --keep-open --udp --exec "/bin/echo lego"'
+
+
+class TestsSpecGiraffe:
+    """A Giraffe spec tests base.
+
+    Args:
+        giraffe: Giraffe instance.
+    """
+
+    _giraffe: Giraffe = None # type: ignore
+
+    @classmethod
+    def setup_class(cls, connections: List[Giraffe]) -> None:
```

Comment:
This is an example that the marking is confusing.
You require the inheriting class to mark.